src/charm.py

```diff
@@ -70,11 +72,12 @@ def __init__(self, *args):
 
         self._stored.set_default(binary_url=self.config.get('binary-url'),
                                  docker_tag=self.config.get('docker-tag'),
-                                 service_args=self.config.get('service-args'))
+                                 service_args=self.config.get('service-args'),
+                                 relay_rpc_urls=dict())
```

Comment:
Due to the data bug in departed hook we need a dict to for each url also store information which unit it is.